back/engines/commercial/project_management/app/policies/project_management/patches/stat_user_policy.rb

```diff
@@ -15,7 +15,7 @@ def resolve_for_active
         def resolve_for_project_moderator
           return scope.none unless user.project_moderator?
 
-          user.moderatable_projects
+          ::UserRoleService.new.moderatable_projects(user)
```

Comment:
This is using the `moderatable_projects` method from the `UserRoleService` instead of the one from the user model (because I wanted to move out this logic from the user model and instead have it in a service)